
class DataException(Exception):
    pass


class DataNotFoundException(DataException):
    pass
